I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00gamemenu.rpy", line 170, in script
    $ ui.interact()
  File "renpy/common/00gamemenu.rpy", line 170, in <module>
    $ ui.interact()
TypeError: 'str' object is not callable

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/common/00gamemenu.rpy", line 170, in script
    $ ui.interact()
  File "D:\1_file\gal\renpy-8.0.3-sdk\renpy\ast.py", line 1131, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "D:\1_file\gal\renpy-8.0.3-sdk\renpy\python.py", line 1061, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/00gamemenu.rpy", line 170, in <module>
    $ ui.interact()
  File "D:\1_file\gal\renpy-8.0.3-sdk\renpy\ui.py", line 299, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "D:\1_file\gal\renpy-8.0.3-sdk\renpy\display\core.py", line 3377, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "D:\1_file\gal\renpy-8.0.3-sdk\renpy\display\core.py", line 4253, in interact_core
    rv = renpy.display.focus.mouse_handler(ev, x, y)
  File "D:\1_file\gal\renpy-8.0.3-sdk\renpy\display\focus.py", line 572, in mouse_handler
    return change_focus(new_focus, default=default)
  File "D:\1_file\gal\renpy-8.0.3-sdk\renpy\display\focus.py", line 528, in change_focus
    rv = widget.focus(default=default)
  File "D:\1_file\gal\renpy-8.0.3-sdk\renpy\display\behavior.py", line 930, in focus
    rv = run(self.hovered)
  File "D:\1_file\gal\renpy-8.0.3-sdk\renpy\display\behavior.py", line 329, in run
    return action(*args, **kwargs)
TypeError: 'str' object is not callable

Windows-10-10.0.22621 AMD64
Ren'Py 8.0.3.22090809
我们之间的爱恋化作少女来谋杀我 1.0
Tue Jan 17 23:11:46 2023
